utils/shareFile.ts

```diff
@@ -0,0 +1,17 @@
+import { Directory, File, Paths } from "expo-file-system";
+import * as Sharing from "expo-sharing";
+
+export async function shareFileFromUrl(url: string) {
+  try {
+    const d = new Directory(Paths.cache, "file_downloads");
+
+    // prevent "Destination already exists" error
+    if (d.exists) d.delete();
+    d.create();
+
+    const file = await File.downloadFileAsync(url, d);
```

Comment:
Haben wir das in https://github.com/evcc-io/app/issues/19 beschriebene Problem, dass hier beim Download die Autorisierung fehlt (entweder Basic Auth und/oder evcc-auth bei den Logs)? Oder bekommt die App hier automatisch Basic Auth und Cookie (evcc login) mit übergeben?

Reply:
Ok, ignoriere meinen Kommentar. Habs gerade getestet. Funktioniert. Nice. Elegante Lösung!